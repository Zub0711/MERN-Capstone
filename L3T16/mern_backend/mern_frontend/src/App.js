import React from "react";
import { Route, Routes } from "react-router-dom";
import Admin from "./components/Adminpage";
import User from "./components/Userpage";
import Home from "./components/Homepage";
import Login from "./components/Login";
import Register from "./components/Register";
import "./App.css";
// I import react,routes,my stylesheet and my components

// I then set my routes and add my components to my app function
function App() {
  return (
    <div className="App">
      <Routes>
        <Route path="/" element={<Home />} />
        <Route path="/Login" element={<Login />} />
        <Route path="/Admin" element={<Admin />} />
        <Route path="/User" element={<User />} />
        <Route path="/Register" element={<Register />} />
      </Routes>
    </div>
  );
}

export default App;
