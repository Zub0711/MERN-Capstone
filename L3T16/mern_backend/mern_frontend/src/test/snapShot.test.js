import React from "react";
import renderer from "react-test-renderer";
import App from "../App";
import { MemoryRouter } from "react-router-dom";

test("snapshot test", () => {
  const tree = renderer
    .create(
      <MemoryRouter>
        <App />
      </MemoryRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
