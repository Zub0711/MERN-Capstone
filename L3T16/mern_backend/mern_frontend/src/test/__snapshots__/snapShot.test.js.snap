// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test 1`] = `
<div
  className="App"
>
  <div>
    <a
      href="/User"
      onClick={[Function]}
    >
      <button>
        View Appointments
      </button>
    </a>
    <a
      href="/Login"
      onClick={[Function]}
    >
      <button>
        Admin Login
      </button>
    </a>
  </div>
</div>
`;
